modules/claims_api/app/controllers/claims_api/v2/veterans/disability_compensation_controller.rb

```diff
@@ -153,13 +153,13 @@ def flashes
 
         def shared_validation
           # Custom validations for 526 submission, we must check this first
-          @disability_compensation_validation_errors = validate_form_526_submission_values!(target_veteran)
+          @claims_api_forms_validation_errors = validate_form_526_submission_values!(target_veteran)
           # JSON validations for 526 submission, will combine with previously captured errors and raise
           validate_json_schema
           # if we get here there were only validations file errors
-          if @disability_compensation_validation_errors
+          if @claims_api_forms_validation_errors
             raise ::ClaimsApi::Common::Exceptions::Lighthouse::JsonDisabilityCompensationValidationError,
-                  @disability_compensation_validation_errors
+                  @claims_api_forms_validation_errors
           end
```

Comment:
This change was made in order to make the error reporting not coupled to 526 specifically, since we want the same in POA now.  In the 526 space all I did was change the variable name the errors go into in order to make it not 526 specific.